DifferentiationInterface/docs/src/api.md

```diff
@@ -139,4 +139,4 @@ The following is not part of the public API.
 Modules = [DifferentiationInterface]
 Public = false
 Filter = t -> !(Symbol(t) in [:outer, :inner])
-```
+```
```

Comment:
Does this do anything? If not, can we remove it to minimize the diffs without meaning?